Modified styling of the copy buttons to be part of the textarea

Wachiwi opened the pull request:
See #13 as main story. 

This currently just uses a different styling for the buttons to integrate them better with the textareas. 

Preview: 
<img width="661" alt="preview" src="https://user-images.githubusercontent.com/4199845/31068246-7ecfd3be-a757-11e7-9211-dbbe72010471.png">


Further I added `*.swp` files to .gitignore for vim users 😉

ducthienbui97 commented:
@Wachiwi  Thanks for your contribution. Your PR is not up-to-date with this repo's master branch. Can you update it?

Wachiwi commented:
@ducthienbui97 Rebased and added toast notification. 

**Important**: I added a new (fixed) dependency for the [toasts](https://apvarun.github.io/toastify-js/). Further they need some more styling.

Wachiwi commented:
Cannot tell why tests fail. They pass locally.

ducthienbui97 commented:
Test failed all the times. It needs specific Github token to run and it's only allowed to read that token in my master branch. Don't worry.

ducthienbui97 commented:
The copied notice is too far away,  can it be near the text areas?
![capture](https://user-images.githubusercontent.com/10867758/31072974-cb9260be-a7b5-11e7-8997-91a311f1971e.PNG)

Wachiwi commented:
Toastify sadly only supports these positioning options: 

```
gravity: "top", // `top` or `bottom`
positionLeft: true // `true` or `false`
```

ducthienbui97 commented:
I actually prefer #20  toastr solution because it uses CDN. But I likes the way you change copy button. I'm thinking about merging #20 then can you modify your code to work with that solution?

Wachiwi commented:
Ok, I'll remove toastify and rebase

Wachiwi commented:
@ducthienbui97 Done

ducthienbui97 commented on public/pages/template.html:
```diff
@@ -15,24 +15,25 @@ <h1 class="cover-heading">
+          <i class="fa fa-copy"></i>
+    </span>
+</div>
+</div>
```
This line should be removed, there are no open tag for this

Wachiwi replied:
Done in 2047181
